src/main/webapp/app/tutor-course-dashboard/tutor-participation-graph/tutor-participation-graph.component.html

```diff
@@ -12,23 +12,16 @@
         >
             <fa-icon [icon]="'chalkboard-teacher'"></fa-icon>
         </li>
-
         <li class="progress-bar-li" [ngClass]="calculateClassProgressBar()">
-            <div class="progress position-relative" [ngbTooltip]="'artemisApp.tutorCourseDashboard.assessStudentsSubmissions' | translate">
-                <div
-                    class="progress-bar"
-                    role="progressbar"
-                    [ngClass]="calculateProgressBarClass()"
-                    [style.width]="percentageAssessmentProgress + '%'"
-                    attr.aria-valuenow="{{ percentageAssessmentProgress }}"
-                    aria-valuemin="0"
-                    aria-valuemax="100"
-                >
-                    <span class="justify-content-center d-flex position-absolute w-100" [ngClass]="chooseProgressBarTextColor()">
-                        {{ numberOfAssessments }} / {{ numberOfParticipations }} ({{ percentageAssessmentProgress + '%' }})
-                    </span>
-                </div>
-            </div>
+            <jhi-progress-bar [percentage]="percentageAssessmentProgress" [numerator]="numberOfAssessments" [denominator]="numberOfParticipations"></jhi-progress-bar>
+        </li>
+        <li class="progress-bar-li" [ngClass]="calculateClassProgressBar()">
+            <jhi-progress-bar
+                [percentage]="percentageComplaintsProgress"
+                [numerator]="numberOfComplaints - numberOfOpenComplaints + (numberOfMoreFeedbackRequests - numberOfOpenMoreFeedbackRequests)"
```

Comment:
Doing calculations within the html template feels a little off to me. Maybe move it into the component?